sawmill-benchmark/src/main/java/io/logz/sawmill/benchmark/SawmillMicroBenchmark.java

```diff
@@ -187,30 +187,10 @@ public void benchmarkPipelineWithVariableRemoveProcessors() {
     }
 
     private Pipeline createPipeline(ExecutionStep... steps) {
-        return createPipeline(false, steps);
-    }
-
-    private Pipeline createPipeline(boolean stopOnFailure, ExecutionStep... steps) {
-        String id = "abc";
-        return new Pipeline(id, Arrays.asList(steps), stopOnFailure);
-    }
-
-    public Doc createDoc(Object... objects) {
-        if (objects.length % 2 != 0) {
-            throw new RuntimeException("Can't construct map out of uneven number of elements");
-        }
-
-        LinkedHashMap map = new LinkedHashMap<>();
-        if (objects != null) {
-            for (int i = 0; i < objects.length; i++) {
-                map.put(objects[i], objects[++i]);
-            }
-        }
-
-        return new Doc(map);
+        return new Pipeline("test-pipeline-id", Arrays.asList(steps), true);
```

Comment:
it was "false" before this commit

Reply:
Makes more sense as true in this context, but has no real impact either way.

Reply:
np, just wanted to make sure it's on purpose